data/json/items/containers.json

```diff
@@ -271,7 +271,7 @@
         "description": "A standard plastic jug used for milk and household cleaning chemicals.",
         "price": 0,
         "weight": 190,
-        "volume": 15,
+        "volume": 18,
```

Comment:
Ah, it turns out the definition of gallon differs between countries. I'll revert that commit.